### PR TITLE
Docs engine: Add Stream to the list of projects

### PR DESCRIPTION
**Pull request by renandincer**

### CODE REVIEW

File: products/docs-engine/src/content/index.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ It’s __open-source__ and [available on GitHub](https://github.com/cloudflare/c
 | [Images](https://developers.cloudflare.com/images)                    | [@cloudflare/cloudflare-docs/.../terraform](https://github.com/cloudflare/cloudflare-docs/tree/master/products/terraform)     |
 | [Terraform](https://developers.cloudflare.com/terraform)              | [@cloudflare/cloudflare-docs/.../images](https://github.com/cloudflare/cloudflare-docs/tree/master/products/images)           |
 | [Workers](https://developers.cloudflare.com/workers)                  | [@cloudflare/cloudflare-docs/.../workers](https://github.com/cloudflare/cloudflare-docs/tree/master/products/workers)         |
+| [Stream](https://developers.cloudflare.com/stream)                  | [@cloudflare/cloudflare-docs/.../stream](https://github.com/cloudflare/cloudflare-docs/tree/master/products/stream)         |
```
Comment (adamschwartz) [inline products/docs-engine/src/content/index.md:32]: Please go alphabetical and make sure `|`s align.

**Review by adamschwartz — CHANGES_REQUESTED** [review]
Thanks for doing this. Please fix the little formatting nits.

File: products/docs-engine/src/content/index.md (hunk 1 above)
Comment (renandincer) [reply products/docs-engine/src/content/index.md:32]: yes!

Comment (renandincer) [reply products/docs-engine/src/content/index.md]: Done! (also a reminder for me to not forget to squash and merge)

File: products/docs-engine/src/content/index.md [hunk 2]
```diff
@@ -27,8 +27,10 @@ It’s __open-source__ and [available on GitHub](https://github.com/cloudflare/c
...
 | [Docs Engine](https://docs-engine-docs.adam.workers.dev) — This site. | [@cloudflare/cloudflare-docs/.../docs-engine](https://github.com/cloudflare/cloudflare-docs/tree/master/products/docs-engine) |
 | [Images](https://developers.cloudflare.com/images)                    | [@cloudflare/cloudflare-docs/.../terraform](https://github.com/cloudflare/cloudflare-docs/tree/master/products/terraform)     |
+| [Stream](https://developers.cloudflare.com/stream)                    | [@cloudflare/cloudflare-docs/.../stream](https://github.com/cloudflare/cloudflare-docs/tree/master/products/stream)           |
 | [Terraform](https://developers.cloudflare.com/terraform)              | [@cloudflare/cloudflare-docs/.../images](https://github.com/cloudflare/cloudflare-docs/tree/master/products/images)           |
 | [Workers](https://developers.cloudflare.com/workers)                  | [@cloudflare/cloudflare-docs/.../workers](https://github.com/cloudflare/cloudflare-docs/tree/master/products/workers)         |
+
```
Comment (adamschwartz) [inline products/docs-engine/src/content/index.md:33]: https://github.com/cloudflare/cloudflare-docs/commit/54c3726279d6a6a728815b0579e16b5a5416f909

**Pull request merged**